### PR TITLE
[FIXED] Alpine entrypoint

### LINKED ISSUE #36: default configuration file is not loaded anymore
Hi,

I think with https://github.com/nats-io/nats-docker/commit/baa13d5bbb48534fa2bd2224e05ad9aa31a36224#diff-d1c0d0786d1f92a5922402e9cc85ca93L31 the default config at `/etc/nats/nats-server.conf` got silently ignored and nats starts with the default configuration.

This is an issue as the provided config file configured `http monitor` by default and many readmes and tutorials assume this behaviour. This is also inconsistent with the current image readme https://github.com/docker-library/docs/tree/master/nats

I also suggest to print the configuration file in use on startup as there is no indication which config file is loaded, even when using `-DVV` flags.

### PR DESCRIPTION
**Pull request by kozlovic**
The previous PR was not the proper way to do it.
Here is some background: https://github.com/docker-library/official-images/pull/8137#issuecomment-639843407

Resolves #36

Signed-off-by: Ivan Kozlovic <ivan@synadia.com>

### CODE REVIEW

**Comment by kozlovic** [comment]
Sorry for another PR, but apparently this is how we should do for non scratch images.
Since there is a shell, it allows this:
```
docker run -ti nats sh
```
or say
```
docker run -ti nats ls
```
But if we start with a `-` flag, then it will be nats-server that is invoked. Say:
```
docker run -ti nats -m 8223
```
See comment link in description for background.

File: 2.1.7/alpine3.11/Dockerfile [hunk 1]
```diff
@@ -26,6 +26,7 @@ RUN set -eux; \
 	apk del --no-cache --no-network buildtmp
 
 COPY nats-server.conf /etc/nats/nats-server.conf
+COPY docker-entrypoint.sh /docker-entrypoint.sh
```
Comment (variadico) [inline 2.1.7/alpine3.11/Dockerfile:29]: I noticed the Postgres Dockerfile puts `docker-entrypoint.sh` in `/usr/local/bin/`.
https://github.com/docker-library/postgres/blob/682ff83c5c83f1b6f2b02caf7aa3e17a491b403a/13/Dockerfile#L185

Totally not important or necessary, but it cleans up the `/` directory a little.

**Comment by variadico** [comment]
Tested:
```
docker run -ti nats sh
docker run -ti nats ls
docker run -ti nats -m 8223
```

Works for me!

**Comment by kozlovic** [comment]
@variadico Hey, thanks for the review and I hope you are doing well!

I have done as you suggested and then also removed the path where I invoked `/usr/local/bin/nats-server` since being in /usr/local/bin it should not be needed. I have tested, but would love if you could test too :-)

**Comment by variadico** [comment]
Just tested, works for me. :+1:

**Pull request merged**